docs/routing.md

```diff
@@ -44,9 +42,8 @@ import { Application, Router } from '@ubio/framework';
 import { StatusRouter } from './routes/status';
 
 export class App extends Application {
-    constructor() {
-        // ...
-        this.bindRouter(StatusRouter);
+    defineHttpRequestScope(mesh: Mesh) {
```

Comment:
Is it more like `addToHttpRequestScope`? Redefining `createHttpRequestScope()` could make it more obvious for the application code reader, app is serving more than just routes mentioned here:
```ts
createHttpRequestScope() {
  super.createHttpRequestScope();
  mesh.service(StatusRouter);
}
```
So, it's now obvious there is something in the scope beside StatusRouter. (e.g. MetricsRouter). WDYT?